### ISSUE TITLE
Undefined index: o:tag includes/wp-mail.php on line 154

### ISSUE THREAD

**Issue by thesenicethings** [mailgun/wordpress-plugin#15]
Notice: Undefined index: o:tag in <wordpress>/plugins/mailgun/includes/wp-mail.php on line 154

Change line 135 to:

``` php
$body = array(
    'from' => "{$from_name} <{$from_email}>",
    'to' => $to,
    'subject' => $subject,
    'text' => $message,
    'o:tag' => ''
);
```

**Comment by tomhoad** [comment]
@travelton Any plans to integrate this into an update?

**Comment by pirogoeth** [comment]
Hi, I've fixed this in #26. Thanks!
